docs/multicluster/user-guide.md

```diff
@@ -533,6 +534,16 @@ metadata:
   namespace: kube-system
 ```
 
+```yaml
+antrea-controller.conf: |
+  featureGates:
+...
+Multicluster: true
+...
+multicluster:
+  enablePodToPodConnectivity: true
```

Comment:
The format is incorrect, please add necessary empty spaces.

Reply:
Yes, I double-checked then found that once I push code, the IDE will format the Markdown file automatically with a wrong format. I enabled the feature now.